flake.nix

```diff
@@ -0,0 +1,23 @@
+{
+  description = "Krops: Lightweight NixOS deployment toolkit";
+
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, nixpkgs, flake-utils }: {
+    lib = import ./lib {
+      nixpkgsLib = nixpkgs.lib;
+    };
+  } // (flake-utils.lib.eachSystem flake-utils.lib.allSystems (system:
+    let
+      pkgs = nixpkgs.legacyPackages.${system};
+      populate = pkgs.callPackage ./pkgs/populate {
+        inherit (self) lib;
+      };
+    in {
+      packages = (pkgs.callPackage ./pkgs/krops {
```

Comment:
We flakes you can build stuff like:

```
$ nix run 'github:Mic92/dotfiles#deploy.eve`
```

https://github.com/Mic92/dotfiles/blob/master/flake.nix#L55
https://github.com/Mic92/dotfiles/blob/master/nixos/krops.nix#L18